package.json

```diff
@@ -34,7 +34,7 @@
     "@svitejs/changesets-changelog-github-compact": "^1.1.0",
     "esbuild": "^0.19.5",
     "esno": "^4.0.0",
-    "gl": "^6.0.2",
+    "gl": "^8.0.2",
```

Comment:
was this causing an issue in Vercel or for you locally?

Reply:
locally - couldn't `pnpm i` without this